### ISSUE TITLE
change the default time horizon

### ISSUE THREAD

**Issue by livehop** [guiqui/react-timeline-gantt#62]
Hi,
Not sure if this is an issue; but is there a way I can change the current default time horizon (it starts from today's date and I wanted it to start from today's date minus X days  ...

**Comment by bustEXZ** [comment]
Yes, i need the same. Some think like start from data[0].start date.... How can we do that?

**Comment by rosanzheng** [comment]
Would be great to get an answer here! Struggling with the same problems.
